components/roadmap/RoadmapItem.js

```diff
@@ -9,7 +10,7 @@ export default function RoadmapItem({
   collapsed = true,
 }) {
   const [isCollapsed, setIsCollapsed] = useState(collapsed)
-  const isCollapsible = item.solutions || item.bets
+  const isCollapsible = item.solutions || item.implementations
```

Comment:
Since we're not using the Shape Up framework anymore, I'm calling this `implementations`, which are a combination of repos, issues, or pull requests.